gulpfile.js

```diff
@@ -53,6 +54,18 @@ const js = () => {
     .src('assets/js/*.js')
     .on('error', swallowError)
     .pipe(sourcemaps.init())
+    .pipe(
+      babel({
+        presets: [
+          [
+            '@babel/env',
+            {
+              modules: false
+            }
```

Comment:
```suggestion
              modules: false,
              targets: "> 0.25%, not dead"
            }
```
It doesn't change anything here, but I think it's a reasonable default.